main/getting-started/intro-zoe.md

```diff
@@ -1,358 +1,286 @@
-
-For details about all three APIs and their methods, see the [Zoe API](https://agoric.com/documentation/zoe/api/) documentation.
-
-## Workflow summary
-
-The following table summarizes an Atomic Swap workflow:
-
-<table>
-<tbody>
-<tr>
-<td>1</td>
-<td>Write your contract
-</td>
-</tr>
-<tr>
-<td>2</td>
-<td>Install your contract’s source code on Zoe. 
-</td>
-</tr>
-<tr>
-<td>3</td>
-<td>Start a contract instance based on an installed contract. 
-</td>
-</tr>
-<tr>
-<td colspan="2">At this point, everything is installed and set up and you’ve got a contract instance. The following rows cover what happens while
-the Atomic Swap contract instance is active.
-</td>
-</tr>
-<tr>
-<td>1</td>
-<td>A holder of an invitation to this contract instance uses it to get a seat. The assets used in the offer are escrowed with Zoe.
-</td>
-</tr>
-<tr>
-<td>2</td>
-<td>Likely via means outside of the contract, invitations are distributed to potential users of this contract instance).</td>
-</tr>
-<tr>
-<td>3</td>
-<td>Recipients use the InviteIssuer to claim and validate their invitation.
-</td>
-</tr>
-<tr>
-<td>4</td>
-<td>Invitation holders make their offers.
-</td>
-</tr>
-<tr>
-<td>5</td>
-<td>The contract code processes the offers, which may request some reallocations as a result.</td>
-</tr>
-<tr>
-<td>6</td>
-<td>Zoe reallocates the assets among the offers, preserving offer safety.
-</td>
-</tr>
-<tr>
-<td>7</td>
-<td>The contract requests Zoe process the satisfied offers, resolving their associated payout promises. Offer holders either get what they paid for, get their escrowed offer amount refunded, or in some cases, get both what they paid for and a refund of any overpayment.
-</td>
-</tr>
-</tbody>
-</table>
+<<< @/snippets/test-intro-zoe.js#harden
+
+Now that we have the required pieces, we can make an offer:
+
+<<< @/snippets/test-intro-zoe.js#offer
+
+At this point, Zoe burns our invitation and confirms its validity. Zoe
+also escrows all of our payments and represents their value in amounts
+as our `current allocation` in the contract.
+
+### Obtaining a UserSeat
+
+Making an offer as a user gives back a `UserSeat`. This seat
+represents our position in the ongoing contract. We can use this seat
+to do a number of things, including:
+
+1. exit the contract
+2. get information about our position such as our current allocation
+3. get our payouts from Zoe.
+
+Let's check that our offer was successful:
+
+<<< @/snippets/test-intro-zoe.js#offerResult
+
+The AtomicSwap contract gives back this particular message for our
+offer: "The offer has been accepted. Once the contract has been
+completed, please check your payout." Other contracts and offers may
+return something different. The `offerResult` is entirely up to the
+contract.
+
+### Getting the payouts
+
+Because this was the atomic swap contract, the contract is over once
+the second party escrows the correct assets. Let's get our payout of
+moola using the Keyword we used ('Asset'):
```

Comment:
```suggestion
Because this was an Atomic Swap contract, it is over once
the second party escrows the correct assets to fulfill the offer and do the swap. Get your payout of
moola with the Keyword your used ('Asset'):
```